grunt/tasks/translate/parseSchema.js

```diff
@@ -1,120 +1,112 @@
           var dir = path.parse(filepath).dir;
           var propertiesSchema = grunt.file.readJSON(filepath);
           var key;
           if (item.bowerAttr) {
-            var bower = grunt.file.readJSON(path.join(dir,"bower.json"));
+            var bower = grunt.file.readJSON(path.join(dir, "bower.json"));
             key = bower[item.bowerAttr];
           } else {
             key = path.parse(filepath).name.split(".")[0];
           }
-          
+
           global.translate.schemaData[item.schemaLabel][key] = {};
           global.translate.schemaData[item.schemaLabel][key].properties = propertiesSchema.properties || null;
           global.translate.schemaData[item.schemaLabel][key].globals = propertiesSchema.globals || null;
         });
       });
     }
-    
-    function processGlobals () {
-      [
-        {
-          type: "components",
-          schemaKey: "components", // key to find in global.translate.schemaData
-          schemaLabel: "_components" // name used to save in global.translate.schemaData.models.course
-        },{
-          type: "extensions",
-          schemaKey: "extensions",
-          schemaLabel: "_extensions"
-        },{
-          type: "menu",
-          schemaKey: "menu",
-          schemaLabel: "_menu"
-        }
-      ].forEach(function (item) {
+
+    function processGlobals() {
+      [{
+        type: "components",
+        schemaKey: "components", // key to find in global.translate.schemaData
+        schemaLabel: "_components" // name used to save in global.translate.schemaData.models.course
+      }, {
+        type: "extensions",
+        schemaKey: "extensions",
+        schemaLabel: "_extensions"
+      }, {
+        type: "menu",
+        schemaKey: "menu",
+        schemaLabel: "_menu"
+      }].forEach(function(item) {
         var data = {};
         var collection = global.translate.schemaData[item.schemaKey];
-        
+
         for (var key in collection) {
           var globals = collection[key].globals;
           if (globals !== null) {
-            
+
             var name = key;
             if (item.type === "components") {
-              name = "_"+key;
+              name = "_" + key;
             }
-            
+
             data[name] = {
               type: "object",
               properties: globals
             };
           }
         }
-        
+
         global.translate.schemaData.models.course.properties._globals.properties[item.schemaLabel] = {
           type: "object",
           properties: data
         };
       });
     }
-    
-    function processPluginLocations () {
-      [
-        {
-          type: "components",
-          schemaKey: "components"
-        },{
-          type: "extensions",
-          schemaKey: "extensions"
-        },{
-          type: "menu",
-          schemaKey: "menu"
-        }
-      ].forEach(function (item) {
+
+    function processPluginLocations() {
+      [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?